modules/prebidServerBidAdapter/index.js

```diff
@@ -766,24 +772,33 @@ const OPEN_RTB_PROTOCOL = {
       }
     }
 
-    if (bidRequests && bidRequests[0].gdprConsent) {
-      // note - gdprApplies & consentString may be undefined in certain use-cases for consentManagement module
-      let gdprApplies;
-      if (typeof bidRequests[0].gdprConsent.gdprApplies === 'boolean') {
-        gdprApplies = bidRequests[0].gdprConsent.gdprApplies ? 1 : 0;
-      }
+    if (bidRequests) {
+      if (bidRequests[0].gdprConsent) {
+        // note - gdprApplies & consentString may be undefined in certain use-cases for consentManagement module
+        let gdprApplies;
+        if (typeof bidRequests[0].gdprConsent.gdprApplies === 'boolean') {
+          gdprApplies = bidRequests[0].gdprConsent.gdprApplies ? 1 : 0;
+        }
 
-      if (request.regs) {
-        if (request.regs.ext) {
-          request.regs.ext.gdpr = gdprApplies;
+        if (request.regs) {
```

Comment:
can replace line numbers  783-790, with a single line:
```
utils.deepSetValue(request, 'regs.ext.gdpr', gdprApplies);
```

Reply:
this was existing code.  just the indentation changed.